foca/utils/logging.py

```diff
@@ -0,0 +1,78 @@
+"""Decorator for logging requests and responses."""
+
+import logging
+from connexion import request
+from functools import wraps
+from typing import (Callable, Optional, )
+
+logger = logging.getLogger(__name__)
+
+
+def log_traffic(_fn: Callable = None,
+                log_request: Optional[bool] = True,
+                log_response: Optional[bool] = True,
+                log_level: Optional[int] = 20,
+                ) -> Callable:
+    """Customizable decorator for logging requests and responses on
+    configurable log level.
+
+    Args:
+        _fn: The function to be decorated (gets passed implicitly).
+        log_request: A boolean flag to set whether or not the request should
```

Comment:
No need to mention types and defaults, this is obvious from the function's signature.